packages/block-editor/src/components/iframe/use-scale-canvas.js

```diff
@@ -309,40 +306,15 @@ export function useScaleCanvas( {
 
 	const previousIsZoomedOut = useRef( false );
 
-	/**
-	 * Runs when zoom out mode is toggled, and sets the startAnimation flag
-	 * so the animation will start when the next useEffect runs. We _only_
-	 * want to animate when the zoom out mode is toggled, not when the scale
-	 * changes due to the container resizing.
-	 */
-	useEffect( () => {
-		const trigger =
-			iframeDocument && previousIsZoomedOut.current !== isZoomedOut;
-
-		previousIsZoomedOut.current = isZoomedOut;
-
-		if ( ! trigger ) {
-			return;
-		}
-
-		startAnimationRef.current = true;
-
-		if ( ! isZoomedOut ) {
-			return;
-		}
```

Comment:
I do like the early return to show that we're not doing anything if we're not zoomed out. Could we add this to the combined `useEffect` to make it clearer it doesn't do anything if we're not zoomed out?